src/ramses_tx/ramses.py

```diff
@@ -278,9 +278,10 @@
     Code._12A0: {  # indoor_humidity
         # .I --- 32:168090 --:------ 32:168090 12A0 006 0030093504A8
         # .I --- 32:132125 --:------ 32:132125 12A0 007 003107B67FFF00  # only dev_id with 007
+        # .I --- 37:153226 --:------ 37:153226 12A0 021 003108127FFF00 01EF7FFF7FFF0002 420615 03AC 00 # ClimaRad VenturaV1x
         # RP --- 20:008749 18:142609 --:------ 12A0 002 00EF
         SZ_NAME: "indoor_humidity",
-        I_: r"^00[0-9A-F]{2}([0-9A-F]{8}(00)?)?$",
+        I_: r"^00[0-9A-F]{2}([0-9A-F]{8}(00)?)?([0-9A-F]{26}(00))?$",
```

Comment:
```suggestion
        I_: r"^00[0-9A-F]{2}([0-9A-F]{8}(00)?)?",
```

Reply:
IIRC, `I` pkt payloads can be arrays, but `RP` not.

Reply:
Your suggested parse results
```
[{'hvac_idx': '00', 'indoor_humidity': 0.51, 'temperature': 20.13, 'dewpoint_temp': None}, {'hvac_idx': '01', 'indoor_humidity': None}, {'hvac_idx': '02', 'indoor_humidity': 0.62, 'temperature': 13.87, 'dewpoint_temp': 6.78}]
```
to me makes little sense for a single HRU unit. Why report 2 "indoor_temperatures" etc?

Reply:
How would Home Assistant understand the results and map them to typical HRU properties?